api/src/main/java/org/pmiops/workbench/monitoring/StackdriverMonitoringService.java

```diff
@@ -0,0 +1,53 @@
+package org.pmiops.workbench.monitoring;
+
+import io.opencensus.exporter.stats.stackdriver.StackdriverStatsExporter;
+import io.opencensus.stats.Aggregation;
+import io.opencensus.stats.Measure;
+import io.opencensus.stats.Stats;
+import io.opencensus.stats.StatsRecorder;
+import io.opencensus.stats.View;
+import io.opencensus.stats.ViewManager;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class StackdriverMonitoringService implements MonitoringService {
+  private static final StatsRecorder STATS_RECORDER = Stats.getStatsRecorder();
+  private static final Logger log = Logger.getLogger(StackdriverMonitoringService.class.getName());
+
+  StackdriverMonitoringService() {
+    try {
+      StackdriverStatsExporter.createAndRegister();
+    } catch (IOException e) {
+      log.log(Level.SEVERE, "Failure to initialize stackdriver stats exporter");
+    }
+  }
+
+  @Override
+  public void registerSignal(
+      String signalName,
+      String signalDescription,
+      Measure measurementInformation,
+      Aggregation aggregation) {
+    View view =
+        View.create(
+            View.Name.create(signalName),
+            signalDescription,
+            measurementInformation,
+            aggregation,
+            Collections.emptyList());
+    ViewManager viewManager = Stats.getViewManager();
```

Comment:
We'll probably want to set up a Singleton Bean to provide the ViewManager so we can mock it in testing.